app/decorators/change_decorator.rb

```diff
@@ -90,14 +90,16 @@ def client_name
   end
 
   def client_phone_number
-    format_phone_number(member.phone_number)
+    decorate_phone_number(member.phone_number)
   end
 
   private
 
-  def format_phone_number(phone_number)
+  def decorate_phone_number(phone_number)
```

Comment:
There is another `format_phone_number` under the application_helper, which was confusing.